ROADMAP.md

```diff
@@ -0,0 +1,25 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: rendering performance, asset management
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
+
+* Merge aframe-core repo into aframe repo
```

Comment:
maybe adapt https://github.com/aframevr/aframe/pull/987/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3 to talk about what 0.2.0 ships